Assets/PatchKit Patcher/Scripts/Utilities/CurrentCultureInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Globalization;
+using System.Linq;
+using UnityEngine;
+
+namespace PatchKit.Unity.Utilities
+{
+    public static class CurrentCultureInfo
+    {
+        public static CultureInfo GetCurrentCultureInfo()
```

Comment:
I would suggest to wrap this method in try...catch and add an fallback mechanism (default culture) - from my experience things like getting system language in .NET might cause problems in some exceptional situations.

Reply:
done